design/ceph/object/store.md

```diff
@@ -113,9 +113,13 @@ This section enables the the object store to be part of a specified ceph-object-
 
 Specifying this section also ensures that the pool section in the ceph-object-zone is used for the object-store. If pools are specified for the object-store they are neither created nor deleted.
 
+Specifying an endpoint in this section means that the endpoint specified will be added to the zones endpoint list. If an endpoint is not specified the endpoint of the object-store the endpoint will be the Kubernetes clusterIP and the port.
```

Comment:
Is there a situation where users would want both the specified endpoint as well as the k8s clusterIP/port info?

Reply:
In most situations where an endpoint (or as I suggest endpoint list) is provided, users would not want the service IPs included.  The endpoints (which are basically rados gateway pods or associated lb) will be presented to the remote cluster as the location to sync data from.  In most cases the service IPs are not going to be reachable by the remote cluster.  If the service IPs are reachable from the remote cluster, then you wouldn't need to specify the endpoints here.

Reply:
@mgugino-uipath ur case in which multiple endpoints need to be given this applies to external ceph cluster, right?

Reply:
I don't think so, the endpoints @mgugino-uipath are the endpoints of the object-stores in the zone. This PR has nothing to do with external ceph clusters, the endpoint(s) being referred to is on the Rook cluster these resources are being created on.

Reply:
If is it is part of same k8s cluster, then we can internal fillup with service rgw name than the custom endpoint. Otherwise it needs to two different k8s cluster or it need to external